src/anyio/_backends/_asyncio.py

```diff
@@ -485,7 +485,7 @@ def __exit__(
             # scope if necessary
             self._restart_cancellation_in_parent()
 
-            # We only swallow the exception iff it was an AnyIO CancelledError, either
+            # We only swallow the exception if it was an AnyIO CancelledError, either
```

Comment:
Why did you change the wording here? "If and only if" is the correct wording, or have I missed something?

Reply:
"iff" means "if and only if"?
I thought it's a misspelling, sorry